covid_xprize/validation/scenario_generator.py

```diff
@@ -58,14 +60,19 @@ def get_raw_data(cache_file, latest=True):
                             encoding="ISO-8859-1",
                             dtype={"RegionName": str,
                                    "RegionCode": str},
-                            error_bad_lines=False)
+                            on_bad_lines='skip')
     latest_df["RegionName"] = latest_df["RegionName"].fillna("")
     # Fill any missing NPIs by assuming they are the same as previous day, or 0 if none is available
-    latest_df.update(latest_df.groupby(['CountryName', 'RegionName'])[NPI_COLUMNS].ffill().fillna(0))
+    latest_df.update(latest_df.groupby(['CountryName', 'RegionName'])[npi_columns].ffill().fillna(0))
     return latest_df
 
 
-def generate_scenario(start_date_str, end_date_str, raw_df, countries=None, scenario="Freeze"):
+def generate_scenario(start_date_str,
+                      end_date_str,
+                      raw_df,
+                      countries=None,
+                      scenario="Freeze",
+                      max_npis_dict=None):
```

Comment:
Additional argument for the dictionary of NPIs (keys) with their max values, to make the script more flexible.